buildSrc/src/main/groovy/Versions.groovy

```diff
@@ -15,6 +15,7 @@ class Versions {
     static String commonsIO = "2.6"
     static String ingestAPI = "0.1.1"
     static String lombok = "1.18.8"
+    static String springActuator = "2.1.6.RELEASE"
```

Comment:
✏️ Dependencies.springActuator() can use the Versions.springBoot instead of needing to add this version here:
```groovy
static def springActuator = { it=Versions.springBoot -> "org.springframework.boot:spring-boot-starter-actuator:${it}" }
```